### PR TITLE
Create BubbleSort.java

### PR DESCRIPTION
**Pull request by Latterization**
for hacktober

### CODE REVIEW

File: Bubblesort/BubbleSort.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+    public static int [] bubbleSort(int [] array){
```
Comment (t2013anurag) [inline Bubblesort/BubbleSort.java]: remove extra spaces.

Comment (t2013anurag) [reply Bubblesort/BubbleSort.java]: fix indentation.

**Comment by Latterization** [comment]
Fixed errors

**Comment by t2013anurag** [comment]
@Latterization - Can you do a small change, and change the directory name and program name to bubble_sort rather than Bubble_sort.

**Comment by Latterization** [comment]
@t2013anurag Yea I can do that

**Comment by Latterization** [comment]
@t2013anurag A director already exists with that name

**Comment by t2013anurag** [comment]
@Latterization - can you move the file inside that directory?

**Comment by Latterization** [comment]
done

**Comment by t2013anurag** [comment]
LGTM, Merging.

**Pull request merged**